mozilla/blurts-server
# Session data

# References: 
Jira: MNTOR-1624



# Description

This adds a `/test/` page that is clearly not finished yet; however, it does take an initial stab at an initial scaffolding for the proper React pages, and more importantly, it shows how we can access user account data in the pages. It also adds a `pendingTranslations.ftl` file, like on Relay.

Also note that this hardcodes a couple of numeric values - I'll bring that up at today's frontend pairing.

# How to test

Visit `/test/` and verify that there's a button to sign in. After signing in, it shows your email address.
